qemu/tests/block_copy.py

```diff
@@ -428,3 +428,44 @@ def verify_md5(self, file_name):
         if status != 0:
             raise error.TestFail("File %s changed, md5sum check output: %s" %
                                  (file_name, output))
+
+    def mount_disks(self):
+        """
+        Mount added disks inside guest.
+        """
+        os_type = self.params["os_type"]
+        if os_type == "linux":
```

Comment:
@CongLi It could be, but I am not sure if now is okay, since format disk part is also rewrote and not stable now. Do we need to discuss with shuang?